webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -78,7 +80,7 @@
                             <span>Extensions</span>
                         </a>
                     </li>
-                    {% if has_telemetry_environment %}
+                    {% if raw.TelemetryEnvironment %}
```

Comment:
I don't think I'd do this. Otherwise the tab is sometimes there and sometimes not and it's weird navigationally-speaking. I think I'd always show it.